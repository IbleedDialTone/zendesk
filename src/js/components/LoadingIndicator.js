import preact from 'preact';

export default () => (
	<span aria-hidden="true" className="loading-animation loading-animation-sm" />
);