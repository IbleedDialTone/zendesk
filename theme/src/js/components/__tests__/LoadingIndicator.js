import preact from 'preact';
import render from 'preact-render-to-string';

import LoadingIndicator from '../LoadingIndicator';

describe('LoadingIndicator', () => {
	it('renders correctly', () => {
		const tree = render(<LoadingIndicator />);

		expect(tree).toMatchSnapshot();
	});
});