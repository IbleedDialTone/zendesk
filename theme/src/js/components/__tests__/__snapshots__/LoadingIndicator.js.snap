// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator renders correctly 1`] = `<span aria-hidden="true" class="loading-animation loading-animation-sm"></span>`;
